web/src/router.rs

```diff
@@ -370,21 +416,32 @@ mod organization_endpoints_tests {
         Ok(())
     }
 
-    // Purpose: creates multiple new Organization instances to a mock DB by calling
-    // the post endpoint supplying the appropriate instance as a JSON payload.
+    // // Purpose: creates multiple new Organization instances to a mock DB by calling
+    // // the post endpoint supplying the appropriate instance as a JSON payload.
```

Comment:
```suggestion
    // the post endpoint supplying the appropriate instance as a JSON payload.
```

Reply:
updated in 3d201b56bb1b97fe95447580f25b3ce9ba5afb7f!